packages/reporters/cli/src/CLIReporter.js

```diff
@@ -216,6 +216,8 @@ async function writeDiagnostic(
   let columns = getTerminalWidth().columns;
   let indent = 2;
   let spaceAfter = isError;
+  diagnostics =
```

Comment:
Easy!

Did you validate this works? Are there any tests we can add a test for this to?

Reply:
Well, the diagnostic is defined in all the cases where it fails since it would have failed earlier in the call stack with a different error to what we see so this should work unless someone is passing something funky... in which case I could add an error log so we can see what the diagnostic is in case it does fail?
In terms of tests, I guess I can copy one of the tests and change the diagnostic to a string.

Reply:
This feels like a strange fix. I think we should either be allowing a Diagnostic to be just a string, or warn in the case that its invalid. This code is handling a scenario that shouldn't really happen.

Reply:
Agree, I think you can add this line to fix the problem.

```js
export function anyToDiagnostic(input: Diagnostifiable): Array<Diagnostic> {
  if (Array.isArray(input)) {
    return input.map(anyToDiagnostic);
  }
  // ...
}
```